util/ui.ts

```diff
@@ -7,8 +7,11 @@ import { Children, isValidElement, cloneElement } from 'react'
  * @param children the child elements to modify.
  * @param newProps the props to add.
  */
-export function renderChildrenWithProps (children, newProps) {
-  const childrenWithProps = Children.map(children, child => {
+export function renderChildrenWithProps(
+  children: JSX.Element[] | JSX.Element,
```

Comment:
Is there an equivalent for type `ReactNode`?

Reply:
See previous comment. Looking at the docs, it looks like this is the preferred way to do it